packages/pyolite-kernel/py/pyolite/pyolite/display.py

```diff
@@ -43,6 +46,11 @@ def format_result(result, raw=False):
     return bundle
 
 
+# TODO Implement the clear_output
+def clear_output(*args, **kwargs):
+    pass
```

Comment:
If you don't mind, I am leaving this for later. It's needed so that ipywidgets can import it. But might not be straightforward to implement.

Reply:
Sounds good :+1:

Reply:
`update_display` will be _really_ fun, too